src/views/Graph.js

```diff
@@ -107,7 +110,7 @@ var GraphView = Backbone.View.extend({
         // Kick off the render loop
         this.animate();
 
-        this.listenTo(events, 'datalasso:data:new', this.redrawEverything);
+        this.listenTo(store, 'change:entries change:mappings change:scales change:attributes change:selectedEntries', this.redrawEverything);
```

Comment:
Yes! I had it on my todo list but forgot. That might cripple things with large datasets.